listenbrainz/db/model/playlist.py

```diff
@@ -80,6 +80,18 @@ class Playlist(BaseModel):
     # mbid of the playlist referred to in copied_from_id
     copied_from_mbid: Optional[uuid.UUID]
 
+    def is_visible_by(self, user_id: int):
+        if self.public:
+            return True
+        if user_id:
+            if user_id == self.creator_id:
+                return True
+            elif user_id in self.collaborator_ids:
+                return True
+            else:
+                return False
+        return False
+
```

Comment:
```suggestion
    def is_visible_to(self, user_id: int):
        if self.public:
            return True
        if user_id and (user_id == self.creator_id or user_id in self.collaborator_ids):
                return True
        return False

```

Reply:
Being a grammar nazi here
Plus a thought for user id test, can it be zero?

Reply:
One thing that I like about the original version is that it's very explicit about what it's doing - 1: check if it's public, 2: check who the creator is, 3: check who the collaborators are. I don't think this change brings any value that makes it significantly better than what was originally there

Reply:
I agree, I think legibility comes first if there is no significant drawback.